ymcatwincities/openy
# Enable the ability to change the CSS for branch locations and camp locations independently

## Steps for review

- Install site.
- Open the configuration page of OpenY Rose or OpenY Lily themes.
- Find block "Custom CSS".
- Add styles according to the manual.
- Check if the relevant pages are changed.

@podarok please review.
